loader/app/poster/post.py

```diff
@@ -100,7 +100,7 @@ def post_doc(db: Session, doc: Document) -> dict:
             "source_id": doc.source_id,
             "geography_id": doc.geography_id,  # this is from backend API lookup, so will exist remotely.
             "category_id": doc.category_id,  # this is from backend API lookup, so will exist remotely.
-            "publication_ts": doc.publication_ts,
+            "publication_ts": doc.publication_ts.isoformat(),
```

Comment:
This is fixed in https://github.com/climatepolicyradar/navigator/pull/554 BTW :+1:

Reply:
the new lines looked like they matched so I thought there would probably be no conflicts

Reply:
:grin: